recipes/airss/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "airss" %}
+{% set version = "0.9.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://www.mtg.msm.cam.ac.uk/files/airss-{{ version }}.tgz 
+    sha256: b54c61b2988b3e6ee6d01e8b73fc60d76f36ad47eeb459d1a0358e486195cc13
+    patches:
+      - patches/makefile.patch
+  - url: https://www.mtg.msm.cam.ac.uk/files/symmol.zip
+    sha256: 5c42701ff54b9c975fe2fff4e9838ef84f14b46a791ff32297558f231ae3e1c3
+    folder: external/symmol
+  - url: https://github.com/spglib/spglib/archive/refs/tags/v1.16.5.zip
+    sha256: fa8f6eb0dc7bb0410f517038e3a165ce74549d19c41a8bdbb34864c7141304a1
+    folder: external/spglib/spglib-1.16.5
+
+build:
+  number: 0
+  skip: true  # [not linux]
```

Comment:
Why can we only build Linux?

Reply:
Upstream only provides support for Linux. Windows is likely to fail because of the outdated Fortran toolchain on cf. I could try to make it work for MacOS.

Reply:
Checking the scripts provided in this package, they are pretty much tailored towards a Linux environment. Porting for MacOS will be non-trivial.